testdata/logging/clusterlogforwarder/fluentd/secure/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "fluentdserver"
+  labels:
+    provider: aosqe
+    component: "fluentdserver"
+    logging-infra: "fluentdserver"
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      provider: aosqe
+      component: "fluentdserver"
+      logging-infra: "fluentdserver"
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        logging-infra: "fluentdserver"
+        provider: aosqe
+        component: "fluentdserver"
+    spec:
+      containers:
+      - name: "fluentdserver"
+        image: "quay.io/openshifttest/fluentd@sha256:7eece00d1bc784ac1e9722b2580911cd3ead5afd740dad6594be945b3b1dd884"
+        imagePullPolicy: "Always"
```

Comment:
Do we need to always pull the image ?

Reply:
No, before using the image digest, I used the `latest` tag and I wanted the image is always the latest, so I set `Allways` here, I can update it to "IfNotPresent".

Reply:
@liangxia updated, PTAL.